content/en/continuous_integration/intelligent_test_runner/python.md

```diff
@@ -57,8 +57,61 @@ After completing setup, run your tests as you normally do:
 DD_ENV=ci DD_SERVICE=my-python-app DD_CIVISIBILITY_ITR_ENABLED=true pytest --ddtrace
 {{< /code-block >}}
 
+## Disabling skipping for specific tests
+
+You can override the Intelligent Test Runner's behavior and prevent specific tests from being skipped. These tests are referred to as unskippable tests.
+
+### Why make tests unskippable?
+
+The Intelligent Test Runner uses code coverage data to determine whether or not tests should be skipped. In some cases, this data may not be sufficient to make this determination.
+
+Examples include:
+
+* Tests that read data from text files
+* Tests that interact with APIs outside of the code being tested (such as remote REST APIs)
+
+Designating tests as unskippable ensures that the Intelligent Test Runner runs them regardless of coverage data.
+
+### Compatibility
+
+Unskippable tests are supported in the following versions and testing frameworks:
+
+* `pytest`
+  * From `ddtrace>=1.19.0`
+
+### Marking tests as unskippable in `pytest`
+
+You can use [`pytest`][2]'s [`skipif` mark][3] to prevent the Intelligent Test Runner from skipping individual tests or modules. Specify the `condition` as `False`, and the `reason` as `"datadog_itr_unskippable"`.
+
+#### Individual tests
+
+Individual tests can be marked as unskippable using the `@pytest.mark.skipif` decorator as follows:
+```
```

Comment:
this is not currently syntax highlighted it looks like (see [link](https://docs-staging.datadoghq.com/romain.komorn/CIAPP-7624-intelligent_test_runner_pytest_unskippable_docs/continuous_integration/intelligent_test_runner/python/?tab=onpremisesciproviderdatadogagent#individual-tests)). I think this should do the trick: 
```suggestion
```python
```